doc/build-unix.md

```diff
@@ -82,12 +82,12 @@ Now, you can either build from self-compiled [depends](/depends/README.md) or in
 
     sudo apt-get install libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev
 
-BerkeleyDB is required for the wallet.
+BerkeleyDB is required for the wallet. Releases ship with BerkeleyDB 4.8, but any version from 4.7 to 5.3 can be used with minimal compatibility issues.
 
-Ubuntu and Debian have their own `libdb-dev` and `libdb++-dev` packages, but these will install
-BerkeleyDB 5.1 or later. This will break binary wallet compatibility with the distributed executables, which
-are based on BerkeleyDB 4.8. If you do not care about wallet compatibility,
-pass `--with-incompatible-bdb` to configure.
+Ubuntu and Debian have their own `libdb-dev` and `libdb++-dev` packages, which install BerkeleyDB 5.3.
+
+If you use BerkeleyDB 6.0 or later, the database file will be incompatible with the distributed executables.
```

Comment:
Add ...`" and you will be legally required to publish the source code for your node, and modify it to inform peers and RPC clients where to download it."`

Or maybe just say 6.0 isn't supported (and reject it in configure?).

Reply:
(This is due to BDB 6.0+ being only available under the AGPL-3 license)

Reply:
That's a good point. I've dropped `--with--incompatible-bdb` and just made 6.0+ not a valid option.